src/App.tsx

```diff
@@ -1,14 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [selected, setSelected] = useState<Todo | null>(null);
+  const [toDoList, setToDoList] = useState<Todo[]>([]);
+  const [isLoadingTodosList, setIsLoadingTodosList] = useState(true);
+  const [filter, setFilter] = useState('all');
+  const [query, setQuery] = useState('');
+
+  useEffect(() => {
+    setIsLoadingTodosList(true);
+    getTodos()
+      .then(setToDoList)
+      .finally(() => setIsLoadingTodosList(false));
+  }, []);
+
+  function preparedToDoList(
+    list: Todo[],
+    filterType: string,
+    searchQuery: string,
+  ): Todo[] {
+    const copyList = [...list];
+    const normalizedQuery = searchQuery.trim().toLowerCase();
+
+    const filteredByQuery = normalizedQuery
+    ? copyList.filter(todo => 
+      todo.title.toLowerCase().includes(normalizedQuery)
+      )
```

Comment:
Fix this linter errors 
<img width="760" alt="image" src="https://github.com/user-attachments/assets/3b1fca60-51b0-4cfe-801e-1b2bdf760972">